satpy/readers/nwcsaf_nc.py

```diff
@@ -75,21 +82,33 @@ def __init__(self, filename, filename_info, filetype_info):
 
         self.nc = self.nc.rename({'nx': 'x', 'ny': 'y'})
         self.sw_version = self.nc.attrs['source']
-        self.pps = False
```

Comment:
`self.pps` and `self.platform_name` need to be initialized here, even if they are set in an other function.

Reply:
Thx, yes, done

Reply:
I fixed that